build.sbt

```diff
@@ -53,9 +53,12 @@ lazy val hat = project
       case BuildEnv.Stage | BuildEnv.Production =>
         libraryDependencies.value.map(excludeSpecs2)
     }),
+    libraryDependencies += "org.codehaus.janino" % "janino" % "3.1.2",
```

Comment:
intrigued as to why this was added

Reply:
me too :)

Reply:
@dataswifty ?? Probably not a big deal https://github.com/Hub-of-all-Things/HAT2.0/commit/67088343b05f0051c58d142149c9fb8f4f77239c#diff-fdc3abdfd754eeb24090dbd90aeec2ceR56